### PR TITLE
Create Orianda's bio

### PR DESCRIPTION
**Pull request by oriandaleigh**
I want to add my bio  to this page. 

Any questions about what I've written?
Is there anything else you want to know about me?

@DarcyCK

### CODE REVIEW

**Comment by DarcyCK** [comment]
@oriandaleigh  - can you add your favorite color? and perhaps your favorite flower?

File: oriandaleigh-bio.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### oriandaleigh
+####Location
+- Fort Collins
+
```
Comment (DarcyCK) [inline oriandaleigh-bio.md:4]: please add color and flower 🌷

File: oriandaleigh-bio.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+#####Favorite Food
+- Tater Tots!
```
Comment (githubteacher) [inline oriandaleigh-bio.md:6]: That is awesome. 😺 🎉 👽 🐊

**Comment by githubteacher** [comment]
👋

File: oriandaleigh-bio.md (hunk 1 above)
Comment (oriandaleigh) [reply oriandaleigh-bio.md:4]: Blue...and lilies!

**Comment by oriandaleigh** [comment]
@DarcyCK does that answer your question? OK to ship?

**Pull request merged**